### DISCUSSION TITLE
如何屏蔽直播“天选时刻”弹窗

### DISCUSSION THREAD

**Discussion by wyq977** [the1812/Bilibili-Evolved#3508] [提问 / Q&A]
Hi，我搜了好多教程但是并没有发现如何屏蔽直播弹窗。

下图是ipad截图，但是网页端也长得一样。

![](https://i0.hdslb.com/bfs/article/565d95a6a1afa3d03a2d1a27bf42f93405bc4f4d.png@942w_1257h_progressive.webp)

广告屏蔽插件如AdGuard, uBlock Origin也没办法屏蔽。

请问插件和其他方法有可以屏蔽的吗？

谢谢！

**Comment by the1812** [comment]
目前没有支持屏蔽这个 (比较难遇到

**Reply by wyq977** [reply]
不知道你有没有常看直播 我看LPL直播每一场结束都会有这个弹窗有点烦。。。
